recipes/emi/meta.yaml

```
@@ -0,0 +1,41 @@
+{% set name = "emi" %}
+{% set version = "0.1.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{version}}
+
+source:
+  url: https://github.com/BioPhys-Turin/EfficientMutualInformation/archive/refs/tags/{{ version }}.tar.gz 
+  sha256: 0551b10cc9e527cff19b7e3c69c293dcf93991c921c9a8cb576050c02d82fb2d
+
+build:
+  number: 0
+  skip:
+    - win
+    - osx
```

This syntax is not correct. It should be:
```suggestion
  skip:  True # [win or osx]
```